build/build.yml

```diff
@@ -24,7 +24,7 @@ steps:
   condition: eq('${{ parameters.yum_install_python }}', 'false')
   displayName: Use Python 3 ${{ parameters.python_arch }}
   inputs:
-    versionSpec: '3.x'
+    versionSpec: '3.11'
```

Comment:
Python 3.12 has been released, in the past we used 3.11 
Python 3.12 no longer works as some packages like disutils are no longer installed by default
Will fix that seaprately